File: Java/compareArray.java
import java.util.Scanner;

public class compareArray {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter a size of an Array 1: ");
        int n1 = sc.nextInt();
        int arr1[] = new int[n1];
        System.out.println("Enter an elements in Array 1: ");
        for (int i = 0; i < n1; i++) {
            arr1[i] = sc.nextInt();
        }

        System.out.println("Enter a size of an Array 2: ");
        int n2 = sc.nextInt();
        int arr2[] = new int[n2];
        System.out.println("Enter an elements in Array 2: ");
        for (int i = 0; i < n2; i++) {
            arr2[i] = sc.nextInt();
        }
        System.out.print("\nElements not in Array 1: ");
        for (int i = 0; i < n2; i++) {
            boolean diff = false;
            for (int j = 0; j < n1; j++) {
                if(arr2[i] == arr1[j]){
                    diff = true;
                }
            }
            if(!diff){
                System.out.print(arr2[i] + " ");
            }
        }
        System.out.print("\nElements not in Array 2: ");
        for (int i = 0; i < n1; i++) {
            boolean diff = false;
            for (int j = 0; j < n2; j++) {
                if(arr1[i] == arr2[j]){
                    diff = true;
                }
            }
            if(!diff){
                System.out.print(arr1[i] + " ");
            }
        }
    }
}

File: java_to_js/compareArray.js
function compareArray(n1,n2){
    let arr1 = n1.split(" ");
    let arr2 = n2.split(" ");
    let x = "";
    for(let i = 0; i < arr2.length; i++){
        let diff = false;
        for(let j = 0; j < arr1.length; j++){
            if(arr1[j] == arr2[i]){
                diff = true
            }
        }
        if(!diff){
            x += arr2[i] + " "; 
        }
    }

    let y = "";
    for(let i = 0; i < arr1.length; i++){
        let diff = false;
        for(let j = 0; j < arr2.length; j++){
            if(arr1[i] == arr2[j]){
                diff = true;
            }
        }
        if(!diff){
            y += arr1[i] + " ";
        }
    }
    return "Elements not in array 1: " + x + "<br>Elements not in array 2: " + y;
}